### PR TITLE
Add CFP Workshop recap

### PR DESCRIPTION
**Pull request by RaphHaddad**
Add run.bat script

### CODE REVIEW

File: run.bat [hunk 1]
```diff
@@ -0,0 +1 @@
+hugo.exe serve --source "%cd%\src"
```
Comment (aaronpowell) [inline run.bat]: There's already vscode tasks for this, and also, this is only for Windows, what about mac and Linux ;)

File: src/content/posts/cfp-workshop-recap.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+title: "CFP Workshop Recap"
+date: 2019-06-28T00:00:00+10:00
+author: "Raphael Haddad"
```
Comment (aaronpowell) [inline src/content/posts/cfp-workshop-recap.md]: Just a heads up, your name will be pulled from the git commit info

**Review by aaronpowell — COMMENTED** [review]
Is it possible to export the slides as a PDF and include the speaker notes?

File: src/content/posts/cfp-workshop-recap.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+> Overcoming fear, anxiety and doubt as a developer
+
+> How I became brave enough to use Docker
+
+At every conference, there's one or two talks that are personal and come from the heart. Attendees like these talks as they provide a break from technically heavy talks, at the same time, the delivery of talks like this can be hard as the speaker is making themselves vulnerable and revealing a lot of emotion to the audience.
+
+## Conclusion
+
+There are a variety of techniques that are used to make language communicate a message more effectively. When writing titles and abstracts for a submission, it's important to take these into consideration in addition to examining common succesful submission patterns. Be sure to checkout past DDD Sydney agendas to see these common patterns and what works for you!
```
Comment (samnevers) [inline src/content/posts/cfp-workshop-recap.md]: Just a suggestion to make the first sentence of the conclusion concise: "There are a variety of language techniques used to communicate a message effectively."

Comment (RaphHaddad) [reply src/content/posts/cfp-workshop-recap.md]: Will fix

File: src/content/posts/cfp-workshop-recap.md (hunk 2 above)
Comment (RaphHaddad) [reply src/content/posts/cfp-workshop-recap.md]: will fix

File: run.bat (hunk 1 above)
Comment (RaphHaddad) [reply run.bat]: will remove

File: src/content/posts/cfp-workshop-recap.md (hunk 3 above)
Comment (RaphHaddad) [reply src/content/posts/cfp-workshop-recap.md]: i made it "There are a variety of language techniques used to communicate a message more effectively"

**Review by aaronpowell — APPROVED** [review]

**Pull request merged**